src/app/common/directives/oc-carousel/oc-carousel.js

```diff
@@ -2,21 +2,22 @@ angular.module('orderCloud')
     .directive('ocCarousel', ocCarouselDirective)
 ;
 
-function ocCarouselDirective($compile, $templateRequest, $exceptionHandler) {
+function ocCarouselDirective($compile, $templateRequest, $log) {
```

Comment:
Why favor $log over $exceptionHandler? 

Is it possibly because it may be desirable to have a carousel with no display so your aim is to discretely display a log to developer if they are confused but not alert the user in case it was intentional?